graduated/codefresh-run/step.yaml

```diff
@@ -198,6 +204,6 @@ spec:
         - echo "Retrieving logs for build $BUILD_ID"
         - codefresh logs -f $BUILD_ID
       [[- end ]]
-        - codefresh wait --verbose $BUILD_ID
+        - codefresh wait --verbose $BUILD_ID --timeout (printf "%s" .Arguments.TIMEOUT_MINS)
```

Comment:
printf is a golang templates command so I think [[ ]] brackets is also required here.
`--timeout [[ (printf "%s" .Arguments.TIMEOUT_MINS) ]]`

Reply:
Done